onnxruntime/test/python/onnxruntime_test_ort_trainer.py

```diff
@@ -686,36 +686,6 @@ def testBertTrainingGradientAccumulation(self):
         assert_allclose(expected_losses, actual_losses, rtol=rtol, err_msg="loss mismatch")
         assert_allclose(expected_eval_loss, actual_eval_loss, rtol=rtol, err_msg="evaluation loss mismatch")
 
-    def testBertCheckpointingBasic(self):
```

Comment:
Why is this being deleted? This is a valid smoke test for checkpoint

Reply:
This test was initially removed as the functionality was covered in more extensive checkpoint test. However, after an offline discussion, this test is required to test the following sceanrio:
"Loading state into a un-initialized trainer(before running eval/train step) In this case, the state is maintained on a temporary variable until eval/step is executed. However, even before that, the user must be able to access state dict values.     

testToyBertCheckpointBasic tests whether the state is there before onnx export

"